components/kyma-environment-broker/internal/process/provisioning/inject_btp_operator_credentials_step.go

```diff
@@ -129,14 +129,20 @@ func updateSecretData(secret *apicorev1.Secret, secretFromParameters *apicorev1.
 
 func (s *InjectBTPOperatorCredentialsStep) createOrRetry(k8sClient client.Client, newSecret *apicorev1.Secret, err error, log logrus.FieldLogger) error {
 	if apierrors.IsNotFound(err) {
+		namespace := &apicorev1.Namespace{ObjectMeta: v1.ObjectMeta{Name: secretNamespace}}
+		err = k8sClient.Create(context.Background(), namespace)
+		if err != nil {
```

Comment:
namespace may already exist, maybe it's worth to handle such a case differently to skip the warning in logs.

Reply:
In the next push.